src/app.js

```diff
@@ -0,0 +1,29 @@
+export default class App {
+  constructor(appElement) {
+    this.appElement = appElement;
+    this.appElement = document.createElement('div');
```

Comment:
The `constructor` here can end after `this.appElement = appElement;`. All the App constructor does when you ask for a new "App" is to store what element represents the app.